.github/workflows/create_preview_sites.yml

```diff
@@ -0,0 +1,43 @@
+name: Create Preview Sites
+
+on:
+  workflow_call:
+    inputs:
+      APP_ID:
+        description: "App ID for the associated website"
+        required: true
+        type: string
+      APP_NAME:
+        description: "Application name for the comment"
+        required: true
+        type: string
+
+jobs:
+  create_preview_site:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Deploy Site
+      run: |
+        RESPONSE_CODE=$(curl -s -o /dev/null -w "%{http_code}" -X POST \
+             -H "Content-Type: application/json" \
+             -H "X-Hub-Signature: $SIGNATURE" \
+             -d "{\"app_id\": \"$APP_ID\", \"branch_name\": \"${{ github.head_ref }}\"}" \
```

Comment:
Will `github.head_ref` work here? I'm not actually sure what ref it might refer to in this context (maybe master?). I would think you might need to thread the PR git ref through from the "Call Build Preview" via build artifacts (similar to what is described on https://securitylab.github.com/research/github-actions-preventing-pwn-requests/).